docs/xdsl-introduction.ipynb

```diff
@@ -581,7 +581,6 @@
       "  %4 = \"arith.constant\"() {\"value\" = 5 : i32} : () -> i32\n",
       "  %5 = arith.addi %3, %4 : i32\n",
       "  %6 = \"arith.cmpi\"(%2, %5) {\"predicate\" = 4 : i64} : (i32, i32) -> i1\n",
-      "  \"scf.yield\"(%6) : (i1) -> ()\n",
```

Comment:
De we want to have implicit terminators in this notebook? Isn't this confusing for newcomers?

Reply:
With this change the termination is implicit, since we cannot "borrow" `scf.yield` anymore.

I could add a custom `yield` for this notebook, or open an issue to track it and resolve it in another documentation-related PR?

Going through the notebook yesterday, there are a few other things I'd like to fix that seem a bit outdated or come out of the blue, so I'd prefer the latter, but I'm easy.

Reply:
Sure, let's go with a docu-related issue :+1:

Reply:
done: https://github.com/xdslproject/xdsl/issues/1155